src/components/TwoLevelSelector.js

```diff
@@ -0,0 +1,377 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const Radium = require('radium');
+
+const Icon = require('./Icon');
+const StyleConstants = require('../constants/Style');
+
+const TwoLevelSelector = React.createClass({
+
+  propTypes: {
+    data: React.PropTypes.object,
```

Comment:
I think the expected data structure can be solidified a bit more. Here is what I propose:
```javascript
[
  {
    label: 'Has A Second Level Menu', //optional: if not set, just display the value
    value: 'menu-1',
    items: [
      {
        label: 'Second Level Menu Item', //optional: if not set, just display the value
        value: 'second-level-1',
      }
    ],
  },
  {
    label: 'No Second Level Menu', //optional: if not set, just display the value
    value: 'menu-2'
  }
]
```